src/v/wasm/wasi.cc

```diff
@@ -306,21 +252,21 @@ errno_t preview1_module::fd_write(
   ffi::memory* mem, fd_t fd, ffi::array<iovec_t> iovecs, uint32_t* written) {
     if (fd == 1 || fd == 2) {
         uint32_t amt = 0;
-        auto logger = fd == 1 ? &_stdout_log_writer : &_stderr_log_writer;
-        for (const iovec_t& vec : iovecs) {
-            try {
-                ffi::array<char> data = mem->translate_array<char>(
+        auto level = fd == 1 ? ss::log_level::info : ss::log_level::warn;
```

Comment:
@rockwotj - Thanks for sorting this out btw. I spent a few hours staring at it while you were out and couldn't quite visualize where things ought to go.